### DISCUSSION TITLE
Replay proxy like ffuf

### DISCUSSION THREAD

**Discussion by brenocss** [projectdiscovery/katana#158] [Q&A] [answered]
### Please describe your feature request:

https://github.com/ffuf/ffuf `-replay-proxy       Replay matched requests using this proxy.` 
### Describe the use case of this feature:

Just replay to proxy when match

**Comment by ehsandeep** [comment]
> Just replay to proxy when match

by match, do you mean results? which can be already send to burp with `-proxy http://127.0.0.1:8080` or could you clarify this more?

**Comment by parthmalhotra** [comment]
@brenocss : FFUF replay only posts found paths to a forward proxy/burp, The crawler doesn't work this way. The crawler visits all paths/urls found, So I don't think the -replay-proxy feature will be needed in katana, as @ehsandeep mentioned you can always use the -proxy option here.

**Comment by brenocss** [comment]
@parthmalhotra @ehsandeep -proxy will pass all http requests (javascript page, css, and all crawled and scanned pages) http through burp, which will make the crawling process slow and blocking (waiting for the proxy to make the request and forward it to the katana). Replay proxy will only send crawled pages respecting the filters entered by the user such as match extensions to burp, and will not wait for the response( just to save the page in burp )
```
katana -u example.com -em php,html -replay-proxy http://127.0.0.1:8080
```

**Comment by parthmalhotra** [comment, marked as answer]
@brenocss : Yes this can be implemented in future to be used with different filters, In the meantime you can easily achieve this by piping the filtered katana output to httpx

```
katana -u example.com -em php,html | httpx -proxy http://127.0.0.1:8080
```

**Reply by brenocss** [reply]
can we convert this to a issue/feature request? @parthmalhotra @ehsandeep

**Reply by MetzinAround** [reply]
I'd say go ahead and open it as a feature request at https://www.github.com/projectdiscovery/katana/issues/new/choose
